packages/app/src/offscreen/index.ts

```diff
@@ -0,0 +1,3 @@
+function isTrueOffscreen() {
```

Comment:
Why do you need this function without export here?

Reply:
It's just for this PR, next PR will be changed.